src/common/test/componentMocking.tsx

```diff
@@ -81,8 +127,8 @@ export const renderProfileContextWrapper = async (
     });
   };
 
-  const waitForDataChange: TestTools['waitForDataChange'] = async () => {
-    const latest = getLastTime('dataUpdateTime');
+  const waitForDataChange: TestTools['waitForDataChange'] = async previousChangeTime => {
+    const latest = previousChangeTime || getLastTime('dataUpdateTime');
```

Comment:
Minor improvement suggestion to help readability: Could the getLastTime function be defined little closer where it is applied? Seems to be also applied inside submit function, but maybe we could put the definition above this waitForDataChange?

Reply:
Referring to another re-ordering comment:
https://github.com/City-of-Helsinki/open-city-profile-ui/pull/154#discussion_r618158453